README.md

```diff
@@ -564,6 +564,9 @@ IEEE Conf. Computer Vision and Pattern Recognition Workshops (CVPRW), 2021. [You
 55th Annual Conf. on Information Sciences and Systems (CISS), 2021.
 - [Alzugaray, I., Ph.D. Thesis, 2022](#Alzugaray22PhD),  
 *Event-driven Feature Detection and Tracking for Visual SLAM*.
+- <a name="FreemanFAST"></a>Freeman, A., Mayer-Patel, K., Singh, M.,  
```

Comment:
There cannot be two "main" entries. One needs to be the main one and the other, the secondary one. See the example of Bardow CVPR 2016.

Suggestion. Key convention: Freeman23arxiv , as in AuthorYYpublicationvenue